test/src/main/java/org/apache/accumulo/test/conf/PropStoreConfigIT_SimpleSuite.java

```diff
@@ -743,23 +747,44 @@ public void testTablePropInSystemConfigFails() {
                 + " cannot be set at the system or resource group level."
                 + " Set table properties at the namespace or table level."));
       }
+
+      assertEquals(Map.of(), client.instanceOperations().getSystemProperties());
```

Comment:
Isn't this just `assertTrue(map.isEmpty())`? I don't think this needs `assertEquals`

Reply:
Should it not be empty it will give a better error message where it shows the diffs of the two maps.  Seemed like the shortest code to get a good error message.